src/NuGetPackageVerifier/Rules/AssemblyStrongNameRule.cs

```diff
@@ -45,22 +45,23 @@ public IEnumerable<PackageVerifierIssue> Validate(PackageAnalysisContext context
                             if (AssemblyHelpers.IsAssemblyManaged(assemblyPath))
```

Comment:
Could we use that to infer things? That way we wouldn't need to extract files to disk.